src/components/dashboard/Dashboard.tsx

```diff
@@ -1,27 +1,181 @@
+    url: 'https://registration.api.hexlabs.org/applications/',
+    method: 'GET',
+    params: {
+      hexathon: currentHexathon._id,
+      userId: user?.uid
+    }
+  })
 
-  if (profile.permissions?.admin) {
-    return <AdminDashboard profile={profile} />;
+  if (profileLoading || profileError 
+    || branchesLoading || branchesError ) {
+    return <Loading />;
   }
 
-  return <ParticipantDashboard profile={profile} />;
+  return (
+    <Flex
+      flexDir="column"
+      padding={{ base: "0 0 16px", md: "32px 48px" }}
+      margin="auto"
+      maxWidth="1000px"
+    >
+      <Flex
+        flexDir={{ base: "column", md: "row" }}
+        bgGradient={{
+          base: "linear(to-b, #33c2ff, #7b69ec)",
+          md: "linear(to-r, #33c2ff, #7b69ec)",
+        }}
+        boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px"
+        alignItems="center"
+        justifyContent="space-around"
+      >
+        <Box
+          color="white"
+          paddingY="32px"
+          paddingLeft={{ base: "16px", md: "32px" }}
+          paddingRight={{ base: "16px", md: "0px" }}
+        >
+          <Heading size="xl" marginBottom="15px">
+            Welcome Back {profile.name?.first}!
+          </Heading>
+          <Text>
+            We’re happy to see you here! We’re currently running our {currentHexathon.name} and we’d
+            love to see you there!
+          </Text>
+          <HStack
+            maxWidth={{ base: "500px", md: "400px" }}
+            justifyContent="space-around"
+            marginTop={{ base: 4, md: 8 }}
+            spacing={4}
+          >
+            <Button width="45%" color="#7B69EC" borderWidth="1px" borderColor="#7B69EC">
+              Apply
+            </Button>
+            <Button width="45%" bgColor="#7B69EC">
+              Details
+            </Button>
+          </HStack>
+        </Box>
+        {application.confirmed ? (
+        <Box
+          border="8px"
+          borderStyle="solid"
+          borderColor="white"
+          borderRadius="3xl"
+          padding="10px"
+          bgColor="#b4c0fa"
+          marginY={{ md: "20px" }}
+          marginBottom={{ base: "40px" }}
+          marginX={{ base: "auto", md: "64px" }}
+        >
+            <QRCodeSVG
+              value={JSON.stringify({
+                uid: user?.uid,
+                name: {
+                  first: profile.name?.first,
+                  last: profile.name?.last,
+                },
+                email: user?.email,
+              })}
+              bgColor="#b4c0fa"
+              size={140}
+            />
+        </Box>
+        ) : (
+          null
+        )
+        }
+      </Flex>
+      <Stack margin={{ base: "20px", md: 0 }} marginBottom={{ base: 0, md: "15px" }}>
+        <Box margin="35px 25px 15px 25px">
+          <Heading fontSize="36px" fontWeight="semibold" marginBottom="10px">
+            Tracks
+          </Heading>
```

Comment:
Can you change this to be "Application Paths"? Tracks isn't the right term here, though I think this was here from before